examples/imagenet/train_imagenet_multi.py

```diff
@@ -181,6 +197,8 @@ def main():
     val_indices = chainermn.scatter_dataset(val_indices, comm, shuffle=True)
     train_data = train_data.slice[train_indices]
     val_data = val_data.slice[val_indices]
+    train_data = FixedBatchDataset(train_data, args.batchsize)
```

Comment:
Could you introduce a flag which enable FixedBatchDataset?

Reply:
Implemented.

Reply:
I put this flag to be true by default, because I don't want to stop the training process by error after epoch 1 finished.